test/uk/gov/hmrc/emcstfereferencedata/services/RetrieveOtherReferenceDataServiceSpec.scala

```diff
@@ -164,7 +164,123 @@ class RetrieveOtherReferenceDataServiceSpec
       )
         .thenReturn(Future.successful(codeListEntrySeq))
 
-      connector.retrieveTypesOfDocument().map(_ shouldBe typesOfDocumentResult)
+      service.retrieveTypesOfDocument().map(_ shouldBe typesOfDocumentResult)
+    }
+  }
+
+  "RetrieveOtherReferenceDataConnector.retrieveMemberStatesAndCountries" should {
+    "return a successful response" when {
+      "retrieveMemberStates method returns data and retrieveCountries method returns no data" in {
+        val codeListEntrySeq = convertToCrdlCodeListEntrySeq(memberStatesResult)
+
+        when(
+          crdlConnector.fetchCodeList(CodeListCode(equalTo("BC11")), equalTo(None), equalTo(None))(
+            using
+            any(),
+            any()
```

Comment:
anything we can do to avoid the use of any or is it not important in this context?